README.md

```diff
@@ -29,3 +29,6 @@ The extension can also be installed with the following command:
 ```
 code --install-extension /path/to/ngls.vsix
 ```
+## Other Editors
```

Comment:
```suggestion
## Angular Language Service for Other Editors
```

Can you also add a newline above this header?

Reply:
Done